site/src/inspector_panel/DivisionsPanel.css

```diff
@@ -0,0 +1,44 @@
+div.div-panel {
+  width: 350px;
+}
+
+caption.common-props {
+  width: max-content;
+}
+
+caption.other-props {
+  width: max-content;
+}
+
+button.divisions-table {
```

Comment:
There are a lot of copied / cloned .css directives throughout all the theme-specific panel css files- Consider putting all these common rules into a single file called 'ThemePanel.css'. This will make most of the .css you have here (PlacesPanel, BuildingsPanel, TransportationPanel) redundant (and they can be cleaned up a lot / be made much shorter). 

I'm not sure there's any difference between these different files - they all seem to have the same content at first glance. 

For example, the directives for `button.divisions-table` here, and `button.building-table` in the buildings file are identical. They should just come from a single place! 

CSS is extremely expressive- let the cascade do all this work for you! Extract out the common features that don't change from theme to theme and make a 'ThemePanels.css' file. Then change all the current .css from looking for a specific `class-table` class like `divisions-table` and instead call it `theme-table`.
 
if you do need to specifically style certain theme panels a little differently, then you should place only the bits of .css that you need to modify those things on a per-theme basis (that stuff should all go in the theme-specific css files). You can put them to work by adding some class specifiers: 'button.theme-table.places'. 

This means you have a single rule to rule them all via `button.theme-table`, then you can add specific rules for buildings, transportation, etc by adding another class to the mix: `button.theme-table._<themename>_`

Reply:
My initial thought was that it might be useful to have separate files now, since after feedback people want certain properties highlighted from different themes, etc. This was probably taken a bit too far with the styling, but I wanted to keep options open. It was more of a "thinking for the future" choice, but I can definitely clean up.

Reply:
Totally understandable, and it's always good to have an 'idea' of where you want to go with a feature. As my colleague @bhousel is fond of saying, 'Sometimes I have to write and rewrite a feature three times before I get it exactly right'.